test/functional/page_objects/time_picker.ts

```diff
@@ -44,11 +45,26 @@ export function TimePickerProvider({ getService, getPageObjects }: FtrProviderCo
   class TimePicker {
     defaultStartTime = 'Sep 19, 2015 @ 06:31:44.000';
     defaultEndTime = 'Sep 23, 2015 @ 18:31:44.000';
+    defaultStartTimeUTC = '2015-09-18T19:37:13.000Z';
+    defaultEndTimeUTC = 'Sep 23, 2015 @ 18:31:44.000';
 
     async setDefaultAbsoluteRange() {
       await this.setAbsoluteRange(this.defaultStartTime, this.defaultEndTime);
     }
 
+    /**
+     * the provides a quicker way to set the timepicker to the default range, saves a few seconds
+     */
+    async setDefaultAbsoluteRangeViaUiSettings() {
```

Comment:
This should speed go the `before` section of the test, since there is no interaction necessary to set the time range, however it seems it can't be measured in the jenkins stats of a single test run.